tests/lib/config/config-file.js

```diff
@@ -155,6 +156,37 @@ describe("ConfigFile", function() {
 
         });
 
+        it("should throw an error when extends config is not found", function() {
```

Comment:
I know that configs are extremely hard to test, but it's one of the most complicated and important part of ESLint and our coverage for configs/plugins resolution is pretty bad. It feels like we need a lot more tests here.